test/plugin/test_out_forward.rb

```diff
@@ -379,12 +443,18 @@ def create_target_input_driver(do_respond=false, disconnect=false, conf=TARGET_C
     DummyEngineDriver.new(Fluent::ForwardInput) {
       handler_class = Class.new(Fluent::ForwardInput::Handler) { |klass|
         attr_reader :chunk_counter # for checking if received data is successfully deserialized
+        attr_reader :storage
```

Comment:
I found that this name (only for tests) is a bit misleading, because plugins in v0.14 may have "storage plugin" instances.
Could you rename this? Any names are ok... (e.g., `data`, `received`, `content_options`...)